miri.bat

```diff
@@ -2,7 +2,10 @@
 :: Windows will not execute the bash script, and select this.
 @echo off
 set MIRI_SCRIPT_TARGET_DIR=%0\..\miri-script\target
-cargo build %CARGO_EXTRA_FLAGS% -q --target-dir %MIRI_SCRIPT_TARGET_DIR% --manifest-path %0\..\miri-script\Cargo.toml
+
+:: If any other steps are added, the "|| exit /b" must be appended to early
+:: return from the script. If not, it will continue execution.
```

Comment:
Is there nothing like `set -e` in bash, which makes it just do this for *all* commands -- step the script when they fail?

Reply:
From my search earlier, there is not. Only this, or doing essentially the same thing but with a `goto`.

Reply:
Okay... good thing this script is so short...